<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UI</class>
 <widget class="QMainWindow" name="UI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>UI</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="step_btn">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>480</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Step</string>
    </property>
   </widget>
   <widget class="QLabel" name="slower_txt">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>470</y>
      <width>47</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Slower</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="turn_txt">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Turn: 0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="population_txt">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Population: 0 (0%)</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QSlider" name="Time_slider">
    <property name="geometry">
     <rect>
      <x>459</x>
      <y>471</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="play_btn">
    <property name="geometry">
     <rect>
      <x>164</x>
      <y>482</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Play</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>756</width>
      <height>375</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="speed_txt">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>510</y>
      <width>331</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Speed: 0 seconds per turn</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="faster_txt">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>470</y>
      <width>47</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Faster</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="pause_btn">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>480</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Pause</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
